demo/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion "Google Inc.:Google APIs:21"
```

Comment:
Hmm I'd prefer to keep this on the latest API level. Can you set it to '23' here and override these when it's run through Travis?